openmc/geometry.py

```diff
@@ -174,9 +202,9 @@ def get_all_material_universes(self):
         all_universes = self.get_all_universes()
         material_universes = set()
 
-        for universe_id, universe in all_universes.items():
-            cells = universe._cells
-            for cell_id, cell in cells.items():
+        for universe in all_universes:
```

Comment:
If I'm not mistaken, you're iterating over the keys in a dictionary when you intended to iterate over the values?